config/model/classifier.yaml

```diff
@@ -1,3 +1,3 @@
-# @package _group_
+_recursive_: False
 _target_: watchmal.model.classifier.Classifier
-num_classes: 3
+num_classes: 4
```

Comment:
change in option to be documented?

Reply:
Just a change for convenience when choosing this config file. I don't think we should document every option for all the config files that are supposed to be self-documenting, where users should read the config files they are choosing to use.